README.md

```diff
@@ -6,15 +6,15 @@
 [![Build Status](https://travis-ci.com/containous/yaegi.svg?branch=master)](https://travis-ci.com/containous/yaegi)
 [![GoDoc](https://godoc.org/github.com/containous/yaegi?status.svg)](https://godoc.org/github.com/containous/yaegi)
 
-Yaegi is Another Elegant Go Interpreter.
+Yaegi is Yet Another Elegant Go Interpreter.
```

Comment:
The omission of *Yet* is on purpose, it makes a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym) 😄

Reply:
Oh, I see, cool. 😄